README.md

```diff
@@ -222,3 +224,22 @@ import 'rxjs/add/operator/catch';
 
 import './jestGlobalMocks';
 ```
+
+### Allow vendor libraries like jQuery, etc...
```

Comment:
Can we separate this into its own PR? I'd like not to mix unrelated changes in one shot

Reply:
ok I think that's better